### ISSUE TITLE
Don't generated minimized JSON files

### ISSUE THREAD

**Issue by ocornut** [cimgui/cimgui#82]
The json output are minimized (no carriage return, no space) which:
- makes them hard to humanly read (if someone wants to study their structure for parsing)
- hard to patch, hard to follow changes
- makes the git commits very large

I feel this is necessary and the output ,json files should be nicely idented like the LUA files are.

**Comment by sonoro1234** [comment]
The script responsible of json generation is not mine but I will take a look

**Comment by sonoro1234** [comment]
Just pushed commit for that

**Comment by sonoro1234** [comment]
Already indented

**Comment by lmariscal** [comment]
I don't know lua scripting, but other languages have json preffifiers that indent the code correctly.

Example of definitions.json:
```json
{
    "CustomRect_CustomRect":[
        {
            "args":"()",
            "argsT":[],
            "argsoriginal":"()",
            "call_args":"()",
            "cimguiname":"CustomRect_CustomRect",
            "comment":"",
            "constructor":true,
            "defaults":[],
            "funcname":"CustomRect",
            "signature":"()",
            "stname":"CustomRect"}],
    "CustomRect_IsPacked":[
        {
            "args":"(CustomRect* self)",
            "argsT":[
                {
                    "name":"self",
                    "type":"CustomRect*"}],
            "argsoriginal":"()",
            "call_args":"()",
            "cimguiname":"CustomRect_IsPacked",
            "comment":"",
            "defaults":[],
            "funcname":"IsPacked",
            "ret":"bool",
            "signature":"()",
            "stname":"CustomRect"}],
}
```

Prettified json with 2 spaces:
```json
{
  "CustomRect_CustomRect": [
    {
      "args": "()",
      "argsT": [],
      "argsoriginal": "()",
      "call_args": "()",
      "cimguiname": "CustomRect_CustomRect",
      "comment": "",
      "constructor": true,
      "defaults": [],
      "funcname": "CustomRect",
      "signature": "()",
      "stname": "CustomRect"
    }
  ],
  "CustomRect_IsPacked": [
    {
      "args": "(CustomRect* self)",
      "argsT": [
        {
          "name": "self",
          "type": "CustomRect*"
        }
      ],
      "argsoriginal": "()",
      "call_args": "()",
      "cimguiname": "CustomRect_IsPacked",
      "comment": "",
      "defaults": [],
      "funcname": "IsPacked",
      "ret": "bool",
      "signature": "()",
      "stname": "CustomRect"
    }
  ]
}
```

This starts to get a little nitpicky, and I'm sorry for that. But I do agree with ocornut that it improves the readability.
